regen/warnings.pl

```diff
@@ -1476,26 +1487,26 @@ package name.  The C<open> function in the module will display a warning
 
     package MyMod::Abc;
 
-    sub open {
+    sub open2 {
         if (warnings::enabled("deprecated")) {
             warnings::warn("deprecated",
-                           "open is deprecated, use new instead");
+                           "open2 is deprecated, use open instead");
         }
-        new(@_);
+        open(@_);
     }
 
     sub new
     ...
```

Comment:
I don't think the rename adds anything, I think the story the docs are trying to tell is the open() in the original version of the module (above) is being deprecated in favour of new(), which has a stub at the bottom of the chunk.

Reply:
I failed to spot the stub - is removing that stub (on top of the current patch) enough to fix the problem and make the resulting docs coherent?

The next code chunk in the doc suggests that `use warnings FATAL => 'MyMod::Abc';` will fatalize a warning triggered by `MyMod::Abc::open()`. But if `sub open { ... }` has only a "deprecated" warning, that fatalization won't trigger. So something needs to change for the fatalization example to work. (I have not checked, but I'm guessing that the 'deprecated' example was added later, missing the fact that the fatalization example was relying on the original definition of `sub open`.)

Reply:
I think you're right.

The fatalization example could call new() instead, but then I think the stub would need to be expanded to include the old open() implementation.